src/main/resources/config/liquibase/changelog/20250911170100_changelog.xml

```diff
@@ -0,0 +1,117 @@
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="token" type="varchar(32)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+            <column name="jhi_user_id" type="bigint">
+                <constraints nullable="false" unique="true"/>
+            </column>
+        </createTable>
+
+        <addForeignKeyConstraint
+                baseColumnNames="jhi_user_id"
+                baseTableName="calendar_subscription_token_store"
+                constraintName="FK_CALENDAR_SUBSCRIPTION_TOKEN_STORE_ON_USER"
+                deferrable="false"
+                initiallyDeferred="false"
+                onDelete="CASCADE"
+                onUpdate="CASCADE"
+                referencedColumnNames="id"
+                referencedTableName="jhi_user"
+                validate="true"/>
+
+        <!-- Transfer tokens into the new table -->
+        <sql>
+            INSERT INTO calendar_subscription_token_store (token, jhi_user_id)
+            SELECT jhiUser.calendar_subscription_token, jhiUser.id
+            FROM jhi_user jhiUser
+            WHERE jhiUser.calendar_subscription_token IS NOT NULL;
+        </sql>
+
+        <!-- Drop the old column -->
+        <dropColumn tableName="jhi_user" columnName="calendar_subscription_token"/>
+
+        <rollback>
+            <!-- Recreate old user column -->
+            <addColumn tableName="jhi_user">
+                <column name="calendar_subscription_token" type="varchar(32)">
+                    <constraints nullable="true" unique="true"/>
+                </column>
+            </addColumn>
+
+            <!-- Move data back to the user column -->
+            <sql>
+                UPDATE jhi_user
+                SET calendar_subscription_token = (
+                    SELECT store.token
+                    FROM calendar_subscription_token_store store
+                    WHERE store.jhi_user_id = jhi_user.id
+                )
+                WHERE EXISTS (
+                    SELECT 1
+                    FROM calendar_subscription_token_store store
+                    WHERE store.jhi_user_id = jhi_user.id
+                );
+            </sql>
+
+            <!-- Drop new table and constraint -->
+            <dropForeignKeyConstraint baseTableName="calendar_subscription_token_store" constraintName="FK_CALENDAR_SUBSCRIPTION_TOKEN_STORE_ON_USER"/>
+            <dropTable tableName="calendar_subscription_token_store"/>
+        </rollback>
+    </changeSet>
+
+    <changeSet id="20250819203300-create-calendar-subscription-token-store" author="marlonnienaber">
+        <!-- Only execute this changeset if the old calendar_subscription_token column does not exist -->
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="calendar_subscription_token_store"/>
+            </not>
+        </preConditions>
```

Comment:
not needed for prod environments, let's keep the prod environments clean from test server issues